tutorial/tutorial-gabc-02.html

```diff
@@ -61,9 +61,9 @@
 </style>
 <h1>Gregorio Tutorial (part II)</h1>
 
-<p>Here we continue the gabc tutorial that began on <a href="tutorial-gabc-01.html">this page</a>.</p>
+<p>Here we continue the gabc tutorial that began on <a href="./tutorial-gabc-01.html">this page</a>.</p>
 
-<p>The documentation of the gabc notation, the notation we use in this tutorial can be found <a href="/gabc/index.html">here</a>.</p>
+<p>The documentation of the gabc notation, the notation we use in this tutorial can be found <a href="../gabc/index.html">here</a>.</p>
```

Comment:
I think `of` should be changed to `for`, and there should be a comma after the word `tutorial` since that's the end of the clause with starts with the first comma.